deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,52 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+.eerste {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
```

Comment:
Op zich best ok om een align-items center te plaatsen als je bedoeling is geweest om het nog te centreren in de verticale as. Anders is het onnodig.